pathways/simulation.py

```diff
@@ -335,7 +361,10 @@ def run_simulation(
     totals.total_intercepted_pests /= float(num_simulations)
     totals.total_missed_pests /= float(num_simulations)
 
-    return totals
+    if detailed:
+        return details, totals
+    else:
+        return totals
```

Comment:
Please, add documentation of the return value directly to the docstring of this function (`run_simulation`). The details are just for the first simulation while the totals are averages. (It makes sense to not store all that data since the use case is likely visualization of a single run, but it is not necessarily expected.)

Reply:
Obviously, more documentation is always needed. This can be fixed later.